.gitignore

```diff
@@ -23,11 +23,3 @@
 
 # broccoli-debug
 /DEBUG/
-
-# IDEs and editors
-.editorconfig
-.history
-.vscode
-
-# OS generated files
-.DS_Store
```

Comment:
Editor / OS specific files don't belong in a project's gitignore, developers can use their own system wide gitignore file for that.